src/actions.js

```diff
@@ -38,14 +38,14 @@ module.exports = {
             embed: {
                 author: {
                     name: source.message.member.displayName,
-                    icon_url: source.message.avatarURL,
+                    icon_url: source.message.author.displayAvatarURL,
                 },
                 image: {
                     url: (source.message.attachments.first() || {}).proxyURL,
                 },
-                url: source.message.url, // Doesn't appear to work >:[
-                description: source.message.content,
-                timestamp: source.message.createdTimestamp,
+                //url: source.message.url, // Doesn't appear to work >:[
+                description: source.message.content.replace(/([\[\]\(\)])/g, '\\$&')+`\n[Jump to Message](${source.message.url})`,

```

Comment:
Refactor this into it's own function so it's clearer what it's doing:

```suggestion
                description: `${escapeMarkdown(source.message.content)}\n[Jump to Message](${source.message.url})`,

```
```js
const escapeMarkdown = s => s.replace(/([\[\]\(\)])/g, '\\$&')
```